pkg/blockstorage/azure/azuredisk.go

```diff
@@ -7,7 +7,7 @@ import (
 	"strings"
 	"time"
 
-	azcompute "github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2019-07-01/compute"
+	azcompute "github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2019-03-01/compute"
```

Comment:
As per the docs - https://docs.microsoft.com/en-us/azure-stack/user/azure-stack-version-profiles-go?view=azs-2008
`2019-03-01` api version is compatible with the Azure stack as well as public Azure instances. I have also validated that with manual testing